librz/main/rz-ax.c

```diff
@@ -429,7 +429,7 @@ static int rax(RzNum *num, char *str, int len, int last, ut64 *_flags, int *fm)
 					printf("[%d", i);
 				}
 			}
-			if (strbits[i] == '0' && strbits[i - 1] == '1') {
+			if (strbits[i] == '0' && strbits[i - 1] == '1' && i != 0) {
```

Comment:
this check has to be done before actually accessing the memory. check my PR.

Reply:
okay thanks